ebu_tt_live/documents/ebutt3.py

```diff
@@ -377,6 +378,12 @@ def _insert_or_discard(self, document):
         ):
             # This loop goes backwards and checks for trimmed documents
 
+            # We cannot have two documents with the same sequence number
+            if item.document.sequence_number == document.sequence_number:
+                raise SequenceNumberAlreadyUsedError(
+                    ERR_SEQUENCE_NUMBER_ALREADY_USED
+                )
+
             # If any found event is higher sequence number we quit
```

Comment:
The problem of comparing documents raised in #202 might appear here too, I planned on comparing the xml here, so we need to solve the problem in #202 first.